src/WooCommerce.php

```diff
@@ -1509,8 +1511,56 @@ private static function getRepricingOptionsDropdownValues(\WC_Product $product)
       !empty($brand) ? $brand . ' | no repricing' : 'no repricing' => __('No Repricing', Plugin::L10N),
       !empty($brand) ? $brand . ' | lower prices only' : 'lower prices only' => __('Lower prices only', Plugin::L10N),
     ];
-    
+
     return $dropdown_values;
   }
 
+  /**
+   * Calculates shipping price + tax
+   *
+   * @param  object $shipping_method
+   * @return float
+   */
+  public static function getTotalShippingPrice(object $shipping_method):float {
+    $shipping_price = floatval($shipping_method->get_total());
+    $shipping_price_tax = floatval($shipping_method->get_data()['total_tax']);
```

Comment:
There is a method to get the total tax:
https://woocommerce.github.io/code-reference/files/woocommerce-includes-class-wc-order-item-shipping.html#source-view.243